ui/apps/platform/src/Components/PatternFly/ErrorBoundary/ErrorBoundaryPage.tsx

```diff
@@ -112,7 +112,7 @@ function ErrorBoundaryPage({ error, errorInfo }: ErrorBoundaryPageProps): ReactE
                                 flex={{ default: 'flex_1' }}
                             >
                                 <ErrorBoundaryCodeBlock
-                                    code={errorInfo.componentStack}
+                                    code={errorInfo.componentStack ?? ''}
```

Comment:
`errorInfo.componentStack` can now be `null` or `undefined`, but the `code` prop expects a string.